source/installation/upgrade-minio.rst

```diff
@@ -10,67 +10,58 @@ Upgrade a MinIO Deployment
 
-   MinIO **strongly discourages** performing blind updates to production
-   clusters.  You should *always* test upgrades in a lower environment
-   (dev/QA/staging) *before* applying upgrades to production deployments.
+This procedure does **not** require taking downtime and is non-disruptive to ongoing operations.
 
-   Exercise particular caution if upgrading to a :minio-git:`release
-   <minio/releases>` that has backwards breaking changes. MinIO includes
-   warnings in release notes for any version known to not support
-   downgrades.
+This page documents methods for upgrading using the update-than-restart method for both ``systemctl`` and user-managed MinIO deployments.
+Deployments using Ansible, Terraform, or other management tools can use the procedures here as guidance for implementation within the existing automation framework.
+
+.. admonition:: Test Upgrades In a Lower Environment
+   :class: important
 
-   For MinIO deployments that are significantly behind latest stable 
-   (6+ months), consider using 
-   `MinIO SUBNET <https://min.io/pricing?ref=docs>`__ for additional support
-   and guidance during the upgrade procedure.
+   Your unique deployment topology, workload patterns, or overall environment **requires** testing of any MinIO upgrades in a lower environment (Dev/QA/Staging) *before* applying those upgrades to Production deployments, or any other environment containing critical data.
+   Performing "blind" updates to production environments is done at your own risk.
 
-Upgrade Checklist
------------------
+   For MinIO deployments that are significantly behind latest stable (6+ months), consider using |SUBNET| for additional support and guidance during the upgrade procedure.
 
-Review all items on the following checklist before performing an upgrade on
-your MinIO deployments:
+Considerations
+--------------
 
-.. list-table::
-   :stub-columns: 1
-   :widths: 25 75
-   :width: 100%
+Upgrades Are Non-Disruptive
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-   * - Test in Lower Environments
-     - Test all upgrades in a lower environment such as a dedicated 
-       testing, development, or QA deployment.
+MinIO's upgrade-then-restart procedure does *not* require taking downtime or scheduling a maintenance period.
+MinIO restarts are fast, such that restarting all server processes in parallel typically completes in a few seconds. 
+MinIO operations are atomic and strictly consistent, such that applications using MinIO or S3 SDKs can rely on the built-in :aws-docs:`transparent retry <general/latest/gr/api-retries.html>` without further client-side logic.
+This ensures upgrades are non-disruptive to ongoing operations.
 
-       **Never** perform blind upgrades on production deployments.
+"Rolling" or serial "one-at-a-time" upgrade methods do not provide any advantage over the recommended "parallel" procedure, and can introduce unnecessary complexity to the upgrade procedure.
```

Comment:
I'd suggest making this it's own consideration and then having the procedure below the standard procedure, with a link to it here.

Reply:
Based on discussions w/ AB, we don't want to draw more attention to this than absolutely necessary. Plus once I pull this into the platformization docs, this section only applies to the Container platform - for all other platforms, this is not a required block.

For now leaving as-is.